scene/main/node.cpp

```diff
@@ -136,17 +143,19 @@ void Node::_notification(int p_notification) {
 				remove_from_group("_vp_unhandled_key_input" + itos(get_viewport()->get_instance_id()));
 			}
 
-			// Remove from processing first
+			// Remove from processing first.
 			if (_is_any_processing()) {
 				_remove_from_process_thread_group();
 			}
-			// Remove the process group
+			// Remove the process group.
 			if (data.process_thread_group_owner == this) {
 				_remove_process_group();
 			}
 			data.process_thread_group_owner = nullptr;
 			data.process_owner = nullptr;
 
+			data.is_auto_translate_dirty = true;
```

Comment:
Why is it set here if the value is irrelevant outside tree?

Reply:
That was more from my first go at this PR, before placing the new member in `Node`. In `Control`, the `is_rtl_dirty` variable is also set to `true` on exiting (on canvas exiting, in that case).

Reply:
Then `is_rtl_dirty` is probably also set unnecessarily.
I mean, the node won't display any text and you do the same when entering tree, so this line is effectively useless.